docs/manual/manual_stock_led.rst

```diff
@@ -0,0 +1,251 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _manual_led_support:
+
+***********************************************
+Stock LED Support (White and RGB LED Control)
+***********************************************
+All Vertiq Gen 2 modules support use of our LED add-on board available with Vertiq's Pro Kit. For information about purchasing a Pro Kit, or for information about 
+purchasing more LED add-on boards, please contact us at support@vertiq.co. Please refer to your module's family page to see if it belongs to Gen 1 or Gen 2.
+
+Your Pro Kit's LED add-on should include the LED board as well as all wiring necessary to connect with your module.
+
+<PUT A PICTURE OF EVERYTHING THAT COMES IN THE BOX>
+
+Hardware Configuration
+=========================
+Vertiq's Gen 2 modules all feature solder pads that allow you to easily connect with external power sources and other peripherals. The LED peripheral board connects 
+with your module's GPIO2 and GPIO3 connections where GPIO2 controls the RGB LED and GPIO3 controls the white LEDs. Depending on the size of your module, 
+you will see the GPIO values printed as either IOX or GPIOX where X is the GPIO value.
+
+.. list-table:: LED Connection Examples
+   :class: borderless
+   :align: center
+
+   * - .. figure:: ../_static/manual_images/leds/40_led_connections.png
+            :alt: 40-06 LED Connections
+            :align: center
+            :height: 350
+
+            Vertiq 40-06 LED Connections
+     - .. figure:: ../_static/manual_images/leds/81_led_connections.png
+            :alt: Vertiq 81-08 Gen 2 LED Connections
+            :align: center
+            :height: 350
+
+            Vertiq 81-08 Gen 2 LED Connections
+
```

Comment:
This may be just something that will be more clear when we get pictures and actual boards, but what are we supposed to do with the ground for the board? And also I guess its power? Do they just tie into the motors power pads?

Reply:
@vertiq-jordan Maybe worth a double check with Teoli, I suppose he would be the one to know what the final boards will have